BusBoard/BusBoard/tflPrediction.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using Newtonsoft.Json;
+
+namespace BusBoard
+{
+    public class IncomingBusPrediction
+    {
+        [JsonProperty("lineName")] public string LineName { get; set; }
+        [JsonProperty("destinationName")] public string DestinationName { get; set; }
+        [JsonProperty("timeToStation")] public int TimeToArrival { get; set; }
+
```

Comment:
If you swap to using RestSharp rather than JsonConvert you can get ride of the `[JsonProperty]` attributes here